server/index.ts

```diff
@@ -54,11 +54,11 @@ app.get('/*', function (req: Request, res: Response) {
 
 const port = process.env.PORT || 2000;
 app.listen(port, () => {
-  console.log(`🚀 Server is listening at http://localhost:${port}`);
```

Comment:
Any reason for removing this?

Reply:
i thought i put it back ..  my server crashes less when its hard code . i can fix it though